src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,54 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+**New features**
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
+a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS). This ensures the
+prerequisites for installing Submariner on RHOS are satisfied.
```

Comment:
```suggestion
* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS). 
```

Reply:
We have a known issue for this, so cannot use the description that "prerequisites for installing Submariner on RHOS are satisfied". We expect the user to manually configure the SG rules.